protected/humhub/modules/queue/driver/MySQL.php

```diff
@@ -25,7 +25,7 @@ class MySQL extends Queue implements QueueInfoInterface
     /**
      * @inheritdoc
      */
-    public $mutex = 'yii\mutex\MysqlMutex';
+    public $mutex = 'yii\mutex\FileMutex';
```

Comment:
@yurabakhtin Please keep this as it is. I'll change it on client side via local Config file.